src/views/List.jsx

```diff
@@ -1,8 +1,60 @@
 import { ListItem } from '../components';
 import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
 export function List({ data }) {
 	const [searchInput, setSearchInput] = useState('');
+	const navigate = useNavigate();
+
+	const WelcomePrompt = () => {
+		return (
+			<>
+				<h3>Shopping list is currently empty...</h3>
```

Comment:
I support this idea and agree that it will make a better user experience to know what each of those time buckets means. Since it's out of scope for this issue, I just added it as a future "Todo" item on our project board.